blocks.js

```diff
@@ -9584,6 +9705,15 @@ SymbolMorph.prototype.setLabelColor = function (
     this.setColor(textColor);
 };
 
+// SymbolMorph to "scratchblocks"
+
+SymbolMorph.prototype.stringify = function () {
+    var symbol = {
```

Comment:
It's mapping Snap! icon names to scratchblocks icon names.  We might add more icons in the future, I guess.